view/frontend/web/js/pargo.js

```diff
@@ -8,668 +8,626 @@
-          photo: pargoPoint.photo,
-          photo_small: pargoPoint.photo_small,
-
-          save_in_address_book: 0,
-        };
-
-        // selectShippingAddressAction(shippingAddress);
-        // checkoutData.setSelectedShippingAddress(shippingAddress);
-        // checkout.setSelectedShippingRate('pargo_customshipping_pargo_customshipping');
-        // checkout.setShippingAddressFromData(shippingAddress);
-      }
-    } else {
-      pargoDefaultDisplay();
-      if (
-        $(".radio:checked").val() !==
-        "pargo_customshipping_pargo_customshipping"
-      ) {
-        $(".form-shipping-address").show();
-      }
+
+        // todo: remove find chained function
+        $(".form-shipping-address select[name=country_id]")
+            .val("ZA");
+        $(".form-shipping-address " + nameSelector).val(name).trigger('change');
+        $(".form-shipping-address " + lastnameSelector).val(lastname).trigger('change');
+        $(".form-shipping-address input[name=company]").val(point.data.storeName + "-" + point.data.pargoPointCode).trigger('change');
+        $(".form-shipping-address input[name=\"street[0]\"]").val(point.data.address1).trigger('change');
+        $(".form-shipping-address input[name=\"street[1]\"]").val(pargoStreet1).trigger('change');
+        $(".form-shipping-address input[name=\"street[2]\"]").val(pargoStreet2).trigger('change');
+        $(".form-shipping-address input[name=city]").val(point.data.city).trigger('change');
+        $(".form-shipping-address input[name=region]").val(point.data.province).trigger('change');
+        $(".form-shipping-address input[name=postcode]").val(point.data.postalcode).trigger('change');
+        $(".form-shipping-address" + telephoneSelector).val(telephone).trigger('change');
+
+        pargoAlternativeDisplay();
+        /* Custom updates */
+        $("div#checkout-step-shipping_method").css({
+            opacity: "0.5",
+            "pointer-events": "none",
+        });
+
+        setTimeout(function () {
+            $(".radio").each(function () {
+
+                const value = $(this).val();
+
+                if (value === "pargo_customshipping_pargo_customshipping") {
+
+                    const pargoPoint = JSON.parse(localStorage.getItem("pargoPoint"));
+                    /*
+                            var pargoPointTelephone = pargoPoint.phoneNumber.toString();
+                            if(pargoPointTelephone==="")
+                                pargoPointTelephone = "[]";
+
+                    */
+                    let pargoPointTelephone = "";
+                    if(pargoPoint.phoneNumber !== "" && pargoPoint.phoneNumber)
+                    {
+                        pargoPointTelephone = pargoPoint.phoneNumber.toString();
+                    }
+
+                    let pargoStreet1 = pargoPoint.suburb;
+                    let pargoStreet2 = "";
+                    if(pargoPoint.address2 !== "" && pargoPoint.address2)
```

Comment:
Empty string is falsey in JS, so 1st check is not necessary

Reply:
I missed this one, there were many more.